### ISSUE TITLE
Bump to latest netty version. 

### ISSUE THREAD

**Issue by carlmolemans** [aws/aws-sdk-java-v2#2567]
https://github.com/aws/aws-sdk-java-v2/blob/dbae1d1ea46961a32874d5465ee590ed0e2e2559/pom.xml#L102

release notes: https://netty.io/news/2021/05/19/4-1-65-Final.html

**Comment by debora-ito** [comment]
@carlmolemans thank you for reaching out. Are you blocked by any particular feature that was released in netty versioin 4.1.63.Final? I'm asking because I don't see any security risks or CVEs associated with the version upgrade, I'm trying to estimate its priority.

**Comment by carlmolemans** [comment]
Some DNS resolve issues got fixed in 4.1.65.Final.

I can overrule the version specified by you, so there is a workaround, yes. However it would be nice to have them transitively available.

Given that you release on a daily basis I see no reason to hold back on bumping netty to the latest version and enjoy the latest features and bugfixes. 

Furthermore, I note that there are several pull requests made by dependabot on this repository. I wonder why netty is not part of automated version increments? 

In your pom you also state that when one bumps netty one must also bump to the latest tcnative ssl library. The latest version for that is 2.0.40.Final where you guys have 2.0.34.Final configured. 

https://github.com/netty/netty-tcnative/compare/netty-tcnative-parent-2.0.34.Final...netty-tcnative-parent-2.0.40.Final

**Comment by debora-ito** [comment]
Thanks for the follow-up. Release dates are not really the blocking issue, it's prioritization comparing to other tasks we are working on. 
And we changed the dependabot limit to 5 pull requests at a time, that's why netty is not showing yet.

Added this to our backlog.

**Comment by carlmolemans** [comment]
Just out of curiosity, is there a reason why you're limiting dependabot to 5 pull requests?

**Comment by debora-ito** [comment]
We just recently enabled dependabot, and we are upgrading our dependencies one bunch at a time. Some require more work than simply bumping up the version number, for example we run integration tests to make sure there's no change in the expected behavior.

**Comment by carlmolemans** [comment]
@debora-ito Yes I am aware that some bumps require more work due to breaking changes etc., we have a similar setup. I'm looking forward seeing this dependency added to the dependabot :)

**Comment by millems** [comment]
The netty version has been upgraded, and will be released on Mondayish.
